libs/s25main/ingameWindows/iwStatistics.cpp

```diff
@@ -15,10 +15,26 @@
 #include "iwHelp.h"
 #include "network/GameClient.h"
 #include "ogl/FontStyle.h"
+#include "ogl/glFont.h"
 #include "world/GameWorldBase.h"
 #include "world/GameWorldViewer.h"
 #include "gameData/const_gui_ids.h"
 
+namespace {
+
+std::string getPlayerStatus(const GamePlayer& player)
+{
+    const std::string playerId = std::to_string(player.GetPlayerId() + 1);
+    if(player.IsDefeated())
+        return "---";
+    else if(player.isHuman())
+        return "#" + playerId;
```

Comment:
Just to avoid work when not required:
```suggestion
    if(player.IsDefeated())
        return "---";
    else if(player.isHuman())
        return "#" + std::to_string(player.GetPlayerId() + 1);
```

Reply:
You are right its rendering code it should be fast